Promote LoadBalancerIPMode to GA (#47938)
kubernetes/website: content/en/docs/concepts/services-networking/service.md

Reviewer: I would change the heading too:
```diff
-#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+#### Load balancer IP address mode {#load-balancer-ip-mode}
```

{{< feature-state feature_gate_name="LoadBalancerIPMode" >}}

> The [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
> named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
> for a Service with `type` set to `LoadBalancer`. 

Reviewer: ```suggestion
For a Service of `type: LoadBalancer`, a controller can set `.status.loadBalancer.ingress.ipMode`.
```
